packages/starlight/translations/de.json

```
@@ -1,7 +1,7 @@
 {
   "skipLink.label": "Zum Inhalt springen",
   "search.label": "Suchen",
-  "search.shortcutLabel": "(Drücke / zum Suchen)",
+  "search.ctrlKey": "Ctrl",
```

I think this is the only language we should set differently:
```suggestion
  "search.ctrlKey": "Strg",
```
`Ctrl` probably works, but after a bit of research, German keyboards definitely usually have `Strg` on the key, so I think defaulting to this until we hear otherwise makes sense.

---

Makes sense, thanks for the suggestion and review 🙌